App.tsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { AppNavigator } from './navigation/AppNavigator';
```

Comment:
Import path error for AppNavigator. Correct import statement:
import AppNavigator from './src/navigation/AppNavigator';